### PR TITLE
Target minimal TFMs

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @davidfowl

File: src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ namespace Microsoft.Extensions.FileProviders
     public class EmbeddedFileProvider : IFileProvider
     {
         private static readonly char[] _invalidFileNameChars = Path.GetInvalidFileNameChars()
-            .Where(c => c != Path.DirectorySeparatorChar && c != Path.AltDirectorySeparatorChar).ToArray();
+            .Where(c => c != '/' && c != '\\').ToArray();
```
Comment (pranavkm) [inline src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs]: Doing this lets us not cross-compile this project.

**Comment by pranavkm** [comment]
Updated

File: src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs (hunk 1 above)
Comment (rynowak) [inline src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs]: But what if they change the path separator character to :trollface:

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**